zone/bot_database.cpp

```diff
@@ -1360,6 +1362,30 @@ bool BotDatabase::LoadItemBySlot(const uint32 bot_id, const uint32 slot_id, uint
 	return true;
 }
 
+bool BotDatabase::LoadItemSlots(const uint32 bot_id, std::map<uint16, uint32>& m)
+{
+	if (!bot_id) {
+		return false;
+	}
+
+	const auto& l = BotInventoriesRepository::GetWhere(
+		database,
+		fmt::format(
+			"bot_id = {}",
+			bot_id
+		).c_str()
```

Comment:
Not sure this needs to be `c_str()` as repositories take in native strings

Reply:
Oh woops, what I get for copy and pasting the fmt::format from a Message call.